### DISCUSSION TITLE
Blank Settings Screen

### DISCUSSION THREAD

**Discussion by worked** [doudar/SmartSpin2k#469] [Q&A] [answered]
I purchased readymade and it worked last year when I was training over the winter. I haven't touched it in a few months, but tried today. The settings screen is loading white, with no content. The blue light is slowly flashing, so I believe there's connectivity to my wifi (which has not changed, ssid or password). I can access all other pages including the Web Shifter, it will shift with the Web Shifter and Physical Shifter. I also just installed the latest firmware without issue. What do you recommend?

**Comment by eMadman** [comment, marked as answer]
Hmm, I wonder if the file is corrupt.  You can do a firmware reset using the shifters.  

- Disconnect the power from the device
- Press and hold both shifter buttons
- Connect the power
- Wait 2-3 seconds and release the button.

That will bring it back to as fresh firmware state and should get you going.  

If it doesn't work, you can manually save the [settings.html](https://github.com/doudar/SmartSpin2k/raw/develop/data/settings.html) file to your computer and uploading it through the firmware update screen at http://smartspin2k.local/OTAIndex

**Reply by worked** [reply]
The index.html load did not work, but a firmware reset did. Thank you.
